packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -361,18 +368,21 @@ async function launchDesktopApp(app: OfficeApp, manifest: ManifestInfo, document
     throw new ExpectedError(`Sideload to the ${getOfficeAppName(app)} app is not supported.`);
   }
 
-  // for Outlook, open Outlook.exe; for other Office apps, open the document
-  if (app == OfficeApp.Outlook) {
+  // for Outlook, Word, Excel, PowerPoint open {Host}.exe; for other Office apps, open the document
```

Comment:
This comment should be changed back to what was originally there.

Reply:
OK, changed.